tests/base/test_context.py

```diff
@@ -357,6 +357,8 @@ def test_target_prefix(self):
                 reset_context((), argparse_args=AttrDict(name="blarg", func="create"))
                 assert context.target_prefix == join(envs_dirs[0], "blarg")
 
+        reset_context()
```

Comment:
Is there a helper or fixture that automatically resets context?

Reply:
this reset may actually no longer be necessary, let me try to remove it

Reply:
yup, it's no longer needed given the other changes done in `context.__init__`